src/app/shared/tabs/entity-datatable-tab/datatable-single-row/datatable-single-row.component.scss

```
@@ -56,13 +56,17 @@
   letter-spacing: 0.3px;
   font-weight: 500;
   line-height: 1.2;
+  overflow-wrap: anywhere;
+  white-space: normal;
 }
 
 .data-value {
   font-size: 1.0625rem;
   color: rgb(0 0 0 / 87%);
   word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)